### ISSUE TITLE
Segfault on wayland native

### ISSUE THREAD

**Issue by eternal-sorrow** [flathub/com.vscodium.codium#275]
When run on wayland natively (with `--ozone-platform-hint=wayland`) vscodium crashes with segfault. Access to wayland server is allowed with `flatpak override`. When started without this option (through Xwayland), it works well. Version 1.85, but this problem began a couple of versions before that. Earlier it was working well.

**Comment by noonsleeper** [comment]
hey @eternal-sorrow, can you try with

```
flatpak run com.vscodium.codium --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations --ozone-platform=wayland
```
?

**Comment by eternal-sorrow** [comment]
It tured out this microsoft/vscode#184124 issue. The workaround is to set the window border style to custom.

**Comment by noonsleeper** [comment]
Thanks @eternal-sorrow, please keep this issue open to track the upstream issue.

**Comment by mainrs** [comment]
How do you pass the arguments to the flatpak installation? When I launch the program using the app launcher in Gnome, I cannot pass any arguments. And flatseal has no override feature for command line arguments...

**Comment by eternal-sorrow** [comment]
@mainrs to test something I pass them in command line. To set them in permanent, I edit the .desktop file (first copy it to `~.local/share/applications`).

**Comment by mainrs** [comment]
> @mainrs to test something I pass them in command line. To set them in permanent, I edit the .desktop file (first copy it to `~.local/share/applications`).

Aren't those updated when a new flatpak update is published?

**Comment by eternal-sorrow** [comment]
If you copy it to `~.local/share/applications`, nothing will update it.

**Comment by noonsleeper** [comment]
This was fixed on #305, then I'll close this. If you have another problem, please don't hesitate and open a new issue.
